public/app/core/components/sidemenu/TopSection.tsx

```diff
@@ -11,17 +11,33 @@ const TopSection: FC<any> = () => {
     text: 'Search',
     icon: 'search',
   };
-
+  const dbaasLink = {
+    id: 'dbaas',
+    icon: 'database',
+    text: 'DBaaS',
+    url: '/graph/d/pmm-dbaas/pmm-dbaas',
+  };
```

Comment:
Please move this outside the component

Reply:
Will be moved to the proper file (pkg/api/index.go) when we move to step three described on this task.